book/src/SUMMARY.md

```diff
@@ -3,7 +3,6 @@
 [Introduction](./introduction.md)
 - [User guide](./user-guide.md)
   - [Setup](./setup.md)
-    - [Applications](./setup-app.md)
```

Comment:
You probably also want to delete the file?

Reply:
deleted